taurus_pyqtgraph/datainspectormodel.py

```diff
@@ -0,0 +1,205 @@
+from datetime import datetime
+
+from pyqtgraph import TextItem, InfiniteLine
+import numpy as np
+from taurustrendset import TaurusTrendSet
+
+
+class DataInspectorModel(object):
+    """
+    DataInspectorModel is a general class use to point-picking.
+    This class use a numpy library to finding the nearest point
+    from the :class:`pyqtgraph.InfiniteLine`.
+    Attach two widget to the plot:
+        - :class:`pyqtgraph.InfiniteLine` - to selecting point
+        - :class:`pyqtgraph.TextItem` - to showing the x,y value
+    Also this class blink the selected scatter point item.
+    """
+    def __init__(self, plot):
```

Comment:
This is a general comment: as a matter of coding style, we try to keep the **public** API to a minimum in order to facilitate refactorings later on. In parcatice this means that "by default" we make most members (attributes and methods) "protected" by sufixing them with `_` and only leave as "public" those which we explicitly want the users to access.
The philosophy is that it is easier to convert something protected into public if eventually needed than to have to go through a deprecation cycle and leave backwards-compatibility hacks if we want to refactor.

In practice this means that in this class members such as `self.scatter`, `self.informationClouds`, `self.custom_point_style_init`, `self.v_line`, `paintCloud()`, `triggerPoint()`, **etc.** would probably be better as "protected"


For example, in the case of this class

Reply:
the rule in taurus (which IMHO should be followed for taurus_pyqtgraph as well) is to make an exception to PEP8 case recommendation for consistency **in the Qt-related modules**. Any module that is not Qt-related, should follow the general PEP8 case rules. In the case of taurus_pyqtgraph, I think most of it will be Qt-related.

Note that taurus itself is pretty inconsistent since this rule was decided much later than the start of the project, and we decided not to refactor everything. But at least for new code we *try* to follow it

Reply:
That explains a lot, 
Now I see the point.